packages/invoices/src/OutboundShipment/DetailView/Footer/Footer.tsx

```diff
@@ -79,7 +79,7 @@ export const FooterComponent: FC = () => {
                 label={t('button.cancel')}
                 color="secondary"
                 sx={{ fontSize: '12px' }}
-                onClick={() => navigate(-1)}
+                onClick={() => setNavAction('up-one')}
```

Comment:
This is why I exposed this action in a hook outside the Breadcrumb component itself. Cos a straight "back" doesn't quite work if you open an invoice (for example) from an external URL, say. And I figured traversing the breadcrumb / url path is something we'd probably want to do quite a bit?

Reply:
Yup, a very nice fix

Reply:
Even nicer thanks to that "useRegisterAcctions" dependency array! 3244927f 👍